treeherder/perf/alerts.py

```diff
@@ -331,3 +332,48 @@ def _glance_over_retrigger_range(self, data_points_to_retrigger: List[dict]):
         if retrigger_range < self._range_width:
             self.log.warning('Found small backfill range (of size {} instead of {})'
                              .format(retrigger_range, self._range_width))
+
+
+class IdentifyRetriggerableAlerts:
```

Comment:
We could rename this to `IdentifyLatestRetriggerables` or even use the old `IdentifyRetrigerrables` name here.

Reply:
Ok. Will do.